tests/tests_modifications_with_selinux_disabled.yml

```
@@ -0,0 +1,120 @@
+  hosts: all
+  gather_facts: true
+  vars:
+    selinux_all_purge: true
+    selinux_logins_purge: true
+
+  tasks:
+    - name: Ensure SELinux tool semanage
+      package:
+        name:
+          - policycoreutils-python-utils
+        state: present
+      when: ansible_distribution == "Fedora" or
+        (ansible_distribution_major_version | int > 7 and
+         ansible_distribution in ["CentOS", "RedHat", "Rocky"])
+
+    - name: Ensure SELinux tool semanage on EL7
+      package:
+        name:
+          - policycoreutils-python
+        state: present
+      when:
+        - ansible_distribution_major_version | int == 7
+        - ansible_os_family == "RedHat"
+
+    - name: Ensure selinux-policy-targeted
+      package:
+        name:
+          - selinux-policy-targeted
+        state: present
+
+    - name: Add a Linux System Roles SELinux User
+      user:
+        comment: Linux System Roles SELinux User
+        name: sar-user
+    - name: Backup original /etc/selinux/config
+      copy:
+        remote_src: true
+        src: /etc/selinux/config
+        dest: /etc/selinux/config.test_selinux_disabled
+        mode: preserve
+    - name: Upload testing /etc/selinux/config
+      copy:
+        src: selinux.config
+        dest: /etc/selinux/config
+        mode: preserve
+    - name: Switch to permissive to allow login when selinuxfs is not mounted
+      command: setenforce 0
+      changed_when: false
+      when: ansible_selinux.status != "disabled"
+    - name: Get selinuxfs mountpoint
+      command: findmnt -n -t selinuxfs --output=target
+      changed_when: false
+      register: selinux_mountpoint
+    - name: >-
+        Umount selinux mountpoint to emulate SELinux disabled
+        system {{ selinux_mountpoint.stdout }}
+      command: umount -l {{ selinux_mountpoint.stdout }}
+      changed_when: false
+
+    - name: Execute the role and catch errors
+      block:
+        - name: Include role
+          include_role:
+            name: linux-system-roles.selinux
+          vars:
+            # needs https://github.com/ansible-collections/ansible.posix/pull/496
+            # selinux_booleans:
+            #  - {name: 'ssh_sysadm_login', state: 'off', persistent: 'yes'}
+            selinux_fcontexts:
+              - {target: '/tmp/test_dir(/.*)?', setype: 'user_home_dir_t',
+                 ftype: 'd'}
+            selinux_ports:
+              - {ports: '22100', proto: 'tcp', setype: 'ssh_port_t',
+                 state: 'present'}
+            selinux_logins:
+              - {login: 'sar-user', seuser: 'staff_u',
+                 serange: 's0-s0:c0.c1023', state: 'present'}
+
+    - name: >-
+        Mount selinux mountpoint back to
+        system {{ selinux_mountpoint.stdout }}
+      # noqa command-instead-of-module
+      command: >-
+        mount -t selinuxfs selinuxfs {{ selinux_mountpoint.stdout }}
+      changed_when: false
```

```suggestion
      changed_when: true
```
unless you add a `when` condition as mentioned above